content/millennium/r4/scheduling/appointment.md

```diff
@@ -34,6 +34,7 @@ The following fields are returned if valued:
 * [Status](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.status){:target="_blank"}
 * [ServiceCategory](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.serviceCategory){:target="_blank"}
 * [ServiceType](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.serviceType){:target="_blank"}
+* [Slot](https://hl7.org/fhir/r4/appointment-definitions.html#Appointment.slot){:target="_blank"}
```

Comment:
Should it be http instead?

Reply:
Encounter and Patient resources use https. Schedule, Appt and Slot used http. Coverage uses both. I'm not sure why they are not consistent across resources.

Reply:
I believe we are using https on new additions. I asked Max a long time ago about but I have no idea where.